geopandas/tests/test_geom_methods.py

```diff
@@ -672,6 +679,27 @@ def test_centroid_crs_warn(self):
         with pytest.warns(UserWarning, match="Geometry is in a geographic CRS"):
             self.g4.centroid
 
+    @pytest.mark.skipif(
+        not compat.SHAPELY_GE_18,
+        reason="ignore_index keyword introduced in shapely 1.8.0",
+    )
+    def test_make_valid(self):
```

Comment:
Can you also add a check using `is_valid` before and after calling `make_valid`?

Reply:
not sure it's the proper way, but I managed it, let me know if it's what you meant